### ISSUE TITLE
maven.properties resource contains project.build.directory

### ISSUE THREAD

**Issue by mike10004** [mike10004/common-helper#1]
The file `/native-helper/maven.properties` in the built jar has the following contents:

```
project.name = native-helper
project.build.directory = /home/mchaberski/ws/public2/common-helper/native-helper/target
project.version = 4.0.0
project.artifactId = native-helper
project.groupId = com.github.mike10004
```

The `project.build.directory` definition should be removed, for reasons of privacy and build reproducibility.

**Comment by mike10004** [comment]
Addressed in commit 538d0a87e9b45e39a84519a6db2891fbdd30c30a

**Comment by mike10004** [comment]
Resolved as of commit db5285ca387047a27c5dfa3fd170a884eb405484
